openeo-geotrellis/src/test/scala/org/openeo/geotrellis/layers/FileLayerProviderTest.scala

```diff
@@ -986,6 +990,65 @@ class FileLayerProviderTest extends RasterMatchers{
     println(s"Count: $count")
   }
 
+  def unpackTarFile(filePath:Path, outputDir:Path):Unit ={
+    val tarInputStream = new TarArchiveInputStream(new FileInputStream(filePath))
+    try {
+      var entry: TarArchiveEntry = tarInputStream.getNextTarEntry
+      while (entry != null) {
+        val outputFile = new File(outputDir, entry.getName)
+        if (entry.isDirectory) {
+          outputFile.mkdirs()
+        } else {
+          outputFile.getParentFile.mkdirs()
+          val outputStream = new FileOutputStream(outputFile)
+          try {
+
+            val buffer = new Array[Byte](1024)
+            var bytesRead = tarInputStream.read(buffer)
+            while (bytesRead != -1) {
+              outputStream.write(buffer, 0, bytesRead)
+              bytesRead = tarInputStream.read(buffer)
+            }
+          } finally {
+            outputStream.close()
+          }
+        }
+        entry = tarInputStream.getNextTarEntry
+      }
+    } finally {
+      tarInputStream.close()
+    }
+  }
+
+  @Test
+  def testReadKeysToRasterSources(@TempDir tempDir: Path):Unit = {
+    val Filename = "zarrExample.tar"
+    val url = new URL(s"https://artifactory.vgt.vito.be/artifactory/testdata-public/openeo/geotrellis_extrensions/$Filename")
```

Comment:
zarr should normally also work fine over http, could we try that rather than the download?
In the case of this test, we would mostly be reading metadata, so actual data download is very limited.